### PR TITLE
feat: don't build when use master

### PR DESCRIPTION
**Pull request by MatousJobanek**
* a few small changes in the makefiles to use the new functionality provided by https://github.com/codeready-toolchain/toolchain-cicd/pull/44
* don't create e2e ToolchainCluster CR in dev mode

### CODE REVIEW

File: make/run-cicd-script.mk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=MatousJobanek/toolchain-cicd/support-deploy-latest
```
Comment (rajivnathan) [inline make/run-cicd-script.mk:2]: I guess this needs to be reverted?

**Review by rajivnathan — COMMENTED** [review]
Looks good, just 1 question

File: make/run-cicd-script.mk (hunk 1 above)
Comment (MatousJobanek) [reply make/run-cicd-script.mk:2]: yes, sure - this is to make the e2e tests pass and also to verify the change :-)

**Review by alexeykazakov — APPROVED** [review]

File: make/run-cicd-script.mk (hunk 1 above)
Comment (MatousJobanek) [reply make/run-cicd-script.mk]: updated a05a451

**Review by xcoulon — APPROVED** [review]

**Pull request merged**